src/pages/Swap/Components/SwapPlatformSelector.tsx

```diff
@@ -89,7 +89,13 @@ const PriceImpact = ({ priceImpact }: { priceImpact?: Percent }) => {
       fontSize="10px"
       lineHeight="12px"
     >
-      {priceImpact ? (priceImpact.lessThan(ONE_BIPS) ? '<0.01%' : `${priceImpact.toFixed(2)}%`) : '-'}
+      {priceImpact
+        ? priceImpact.equalTo(ZERO)
+          ? '0%'
```

Comment:
**MINOR, non-blocking**
In the Ecorouter we use a "No additional fees" for no fees over the swaps. 

Does it makes sense to place something like "No price impact" here?

Reply:
So, in this case we don't have that much space to display big sentences. The options I was thinking are `-` or `0%`.

Reply:
Let's stick with 0% for now.